sortinghat/core/schema.py

```diff
@@ -136,18 +138,24 @@ class Meta:
         model = Enrollment
 
 
+class AffiliationRecommendationType(graphene.ObjectType):
+    uuid = graphene.String()
+    organizations = graphene.List(graphene.String)
+
+
 class AffiliationResultType(graphene.ObjectType):
     uuid = graphene.String()
     organizations = graphene.List(graphene.String)
 
 
 class JobResultType(graphene.Union):
     class Meta:
-        types = (AffiliationResultType,)
+        types = (AffiliationResultType, AffiliationRecommendationType,)
 
 
 class JobType(graphene.ObjectType):
     job_id = graphene.String()
+    method = graphene.String()
```

Comment:
Done, I have changed this. Just as a comment, I realized this field name matches with the class type name (`jobType`), do you think this could lead to any problem? Thanks!